README.md

```diff
@@ -75,7 +75,8 @@ For provenance and transparency, the PR description must include a link to the o
 
 ## Add a package output to a feedstock
 
-By default, `conda-forge` feedstocks cannot push packages to our channel that another feedstock makes. If you encountered an error 
+By default, `conda-forge` feedstocks cannot push packages to our channel that another feedstock makes. If you encountered an error
 when building your package indicating that the given package was not allowed for your feedstock (e.g., you moved a package
 build from one feedstock to another), you should request the output be added to the new feedstock via this repository. An example request
-is located in [examples/example-add-feedstock-output.yml](examples/example-add-feedstock-output.yml). Make a PR putting your `.yml` request file in the `requests` directory and the `conda-forge/core` team will review it.
+is located in [examples/example-add-feedstock-output.yml](examples/example-add-feedstock-output.yml). You can add both glob patterns
```

Comment:
It would be nice to know what kind of globbing we support. Is this done using the `glob.glob` interface of python? With that information, one can check beforehand (i.e. locally) whether their pattern will work.

Reply:
Done. We use fnmatch.